packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -12,8 +12,8 @@ import 'print.dart';
 /// Signature for [FlutterError.onError] handler.
 typedef FlutterExceptionHandler = void Function(FlutterErrorDetails details);
 
-/// Signature for [FlutterErrorDetails.offendingContext] handler.
-typedef FlutterOffendingContextDescriber = Iterable<DiagnosticsNode> Function(Object offendingContext);
+/// Signature for [DiagnosticPropertiesBuilder] transformer.
+typedef FlutterInformationTransformer = DiagnosticPropertiesBuilder Function(DiagnosticPropertiesBuilder properties);
```

Comment:
Why not 
DiagnosticPropertiesTransformer?
"FlutterInformation" isn't a term I see used elsewhere in the code.

Reply:
Instead of making this in terms of DiagnosticsPropertiesBuilder this could be more simply in terms of
List<DiagnosticsNode>
The extra feature DiagnosticPropertiesBuilder adds to List<DiagnosticsNode> isn't needed in this case.

Reply:
I am struggling on this as well, the DiagnosticPropertiesBuilder has its properties final, I cannot easily replace or create a new DiagnosticPropertiesBuilder with the transformed list. If I really want to do List, I will have to override the DiagnosticableNode.getProperties to do the transform there while keep DiagnosticableNode._builder unchanged, which seems like a hack.